### ISSUE TITLE
Media ID column is Plot Description

### ISSUE THREAD

**Issue by sigpark** [ukdtom/plex2csv.bundle#132]
Running latest Plex Server version 

Version 1.1.4.2757, and column 1 for the Library ID is populated with plot descriptions. tested Movies Export Level 1 and level 2.

Example export CSV's attached as .txt as .csv not supported for upload

[Movies-Level 1-20161023-131257.csv.txt](https://github.com/ukdtom/plex2csv.bundle/files/546322/Movies-Level.1-20161023-131257.csv.txt)
[Movies-Level 2-20161023-130526.csv.txt](https://github.com/ukdtom/plex2csv.bundle/files/546323/Movies-Level.2-20161023-130526.csv.txt)

**Comment by ukdtom** [comment]
Huh....Looks fine to me

![ghi132](https://cloud.githubusercontent.com/assets/4699449/19626280/f462e456-992d-11e6-8928-4084cc125e8d.png)

**Comment by SvenErik1968** [comment]
@sigpark , try turning of the "Wrap lines" in the plex2csv configuration and then do a new export. I know at least Excel does not like the line breaks withing a data line.

**Comment by sigpark** [comment]
Thanks for the help - appears to be a "novice misunderstanding on my behalf. Thanks for the addin.

**Comment by sigpark** [comment]
Hi Tommy,

Thanks very much for reply - sorry I was at my son's football game. I am
looking at the output in MSExcel and Notepadd++ and there appears to be
some "hard" line wrapping causing the issues? Please see screen shots
below. Line 1 is the headre,
Line 2 is the first data line, and lines 3 - 11 is a continuation of the
summary data, then the final line 12 is the Genre?

Any help appreciated, thanks,
Simon

[image: Inline image 1]


[image: Inline image 2]

**Comment by sigpark** [comment]
Thanks for help - will do - Sorry - novice here!

Simon

**Comment by sigpark** [comment]
Sorry - saw subsequent reply to turn off "wrap lines".

Am sure this will resolve

Sorry!

Simon

**Comment by sigpark** [comment]
Hi ukdtom - Tommy,

Any chance you can contact me please?

I would like to share what I have done - it uses plex2csv; I think it is
good, and I think you will agree. Not something I want to post just yet,
but wont be offended if you disagree; that said it was more than 12 hours
work and I believe is stable and parameter driven using your output.

Sorry to be so vague, I won't spam you - if you could send me an email that
I can share the result - even if temp email, so I can set out the
development that I think others would like? Or even acknowledge on this
email and I can post to a dropbox for you?

p.s. I created an incident a week ago - Sorry - but you did respond, as did
SvenErik1968!
Thanks

Simon

**Comment by ukdtom** [comment]
You can contact me in a PM on the forums

https://github.com/ukdtom/plex2csv.bundle/wiki/Troble-Shooting

**Comment by sigpark** [comment]
Thanks Tommy, I will.

I just need to write it up, and I have to take family away for couple of
days - ½ term!

Simon

**Comment by sigpark** [comment]
Hi Tommy / DAne,

I have done tha tand posted two exampel outputs - one limited to 100 lines
and smaller thumbs for size.

Hope you figure out - my explanation is a bit rushed..

Simon

**Comment by sigpark** [comment]
Did this link reach you?

https://drive.google.com/open?id=0B2MfT9IXYT2jRThKeERJWDZPRkt1QThFTkhDRExiNWFLRnVj

**Comment by ukdtom** [comment]
That made it ;-)

**Comment by sigpark** [comment]
Gr8, hope the link works, as the invite was blocked as you know!

Good luck with the debug.

I combine the images I get using wget (from path in plex2csv), with
plex2csv data, and I anchor the images 4points from top and sides of xlsx
cell that zi onset and resize to (this means the images sort and filter
with the text of each media item),  and I insert a hyperlink as one of the
inserted image properties to the content folder (I just open the folder,
rather than launch a Neds player) which I previously copied  to a USB
external drive for some friends. I then protect the worksheet.

Basically you get a portable library, that links to the media folder of the
portable drive.

Let me know when you want me to test something else

Thanks,

Simon
